internal/connor/eq.go

```diff
@@ -66,6 +69,27 @@ func eq(condition, data any) (bool, error) {
 	}
 }
 
+// objectsEqual returns true if the given condition and data
+// contain equal key value pairs.
+func objectsEqual(condition map[string]any, data any) (bool, error) {
+	if data == nil {
+		return condition == nil, nil
```

Comment:
suggestion: Might be worth hitting this case in test.

Reply:
I did add a test for this but I can't tell if its working because codecov seems broken for me.

Reply:
Codecov takes some time for the report to uplpad. Try the local coverage make rule?

Reply:
I updated the test and made sure its covered now